styles.css

```diff
@@ -493,3 +493,265 @@ textarea {
+    background-position: 24px 24px;
+  }
+
+  .desktop-card-details {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 100%;
+    height: 70%;
+    padding-bottom: 80px;
+  }
+
+  .card-title {
+    height: 50px;
+    width: 50%;
+  }
+
+  .work {
+    margin: 0;
+  }
+
+  .card-details {
+    width: 80%;
+    height: 50px;
+  }
+
+  .project-btn {
+    margin-top: 0;
+  }
+
+  .card2 {
+    background-image: url("images/desktop-Snapshoot-Portfolio2.svg");
+    background-repeat: no-repeat;
+
+    /* background-position: right; */
+    background-position: 660px 24px;
+    display: flex;
+
+    /* background-image: none; */
+  }
+
+  .desktop-card-details-left {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 100%;
+    height: 70%;
+    padding: 48px 0 80px 48px;
+  }
+
+  .card-details-left {
+    width: 45%;
+  }
+
+  .card3 {
+    background-image: url("images/desktop-Snapshoot-Portfolio3.svg");
+    background-repeat: no-repeat;
+    background-position: 24px 24px;
+  }
+
+  .card4 {
+    background-image: url("images/desktop-Snapshoot-Portfolio4.svg");
+    background-repeat: no-repeat;
+
+    /* background-position: 24px 24px; */
+    background-position: 660px 24px;
+    display: flex;
+  }
+
+  .about {
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    width: 100%;
+  }
+
+  .about-part {
+    padding-left: 118px;
+  }
+
+  .about-title {
+    margin: 28px 0 0 0;
+  }
+
+  .about-details {
+    width: 450px;
+    height: 100px;
+  }
+
+  .languages-part {
+    padding-right: 142px;
+  }
+
+  .first-skill {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 24px 0 40px;
+  }
+
+  .second-skill {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 24px 0 40px;
+  }
+
+  .third-skill {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 24px 0 40px;
+  }
+
+  .all-languages {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .desktop-langages {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 12px;
+    padding-top: 12px;
+    height: 120px;
+    width: 122px;
+  }
+
+  .desktop-language-text {
+    margin: 12px 0 0 0;
+    padding-right: 20px;
+  }
+
+  .contact-form {
+    grid-column: 1/3;
+    background-image: url("images/contact\ form\ background\ shapes\(1\).svg");
+    width: 98.3%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 40px;
+  }
+
+  .contact-form-details {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .contact-title {
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin: 0;
+  }
+
+  .contact-details {
+    width: 40%;
+  }
+
+  .contact-form-list {
+    margin: 0;
+    width: 450px;
+  }
+
+  input {
+    width: 450px;
+  }
+
+  textarea {
+    width: 450px;
+  }
```

Comment:
- [ ] I think it would be better to put more attention to details, Kindly try to replicate the Figma **inputs** by removing the border-radius also by replicating the padding, and the border color as much as possible🙏 
![image](https://user-images.githubusercontent.com/102757126/215142749-a256d512-1cf8-4af0-a910-3461b78e3b7b.png)
**Figma design**
![image](https://user-images.githubusercontent.com/102757126/215142845-2962af6b-c3d6-4c7e-a976-61d4ada426cf.png)

Reply:
Fixed.